packages/design-system/.ladle/theme/slider.css.ts

```diff
@@ -0,0 +1,145 @@
+import { style } from '@vanilla-extract/css';
```

Comment:
I eyeballed all the styles from the pic, so I doubt they're fully accurate. If anything should be tweaked, I am happy to do so!